src/examples/using-with-cmake/CMakeLists.txt

```diff
@@ -56,6 +56,11 @@ endif()
 # Required to create Conduit's MPI dependencies
 if (ENABLE_MPI)
     find_package(MPI REQUIRED)
+    
+    if(NOT TARGET mpi AND TARGET MPI::MPI_CXX)
```

Comment:
Curious that logic other than find_package(MPI) is needed, if you recall the error case w/o this can you share?

Reply:
Ok, I suspect that is coming from BLT in Axom, not Conduit -- but if think Conduit is the issue  I can look into it.

Reply:
It is definitely coming from Axom.

Reply:
Do you think the entire `if(ENABLE_MPI)` block shouldn't be necessary?

Reply:
outside of this PR.

Reply:
Actually its the `find_package` in the using example i bet.  By default that does not call the config file.